providers/google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceAdapter.java

```diff
@@ -136,11 +136,17 @@
       URI network = URI.create(networks.next());
       assert !networks.hasNext() : "Error: Options should specify only one network";
 
+      Iterator<String> subnetworks = options.getSubnetworks().iterator();
+
+      URI subnetwork = subnetworks.hasNext() ? URI.create(subnetworks.next()) : null;
+      assert !subnetworks.hasNext() : "Error: Options should specify only one subnetwork";
```

Comment:
I made a collection like networks because like networks it can be specified more than once.

I added the blocker to match networks behavior though, but I'll create a separate PR to remove both and bring multiple interfaces (direct or subnetworked) to the module.